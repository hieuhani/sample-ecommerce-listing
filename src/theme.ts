import { DefaultTheme } from 'styled-components'

export const theme: DefaultTheme = {
  colors: {
    reddishOrange: '#f5471e',
    coolGrey: '#8f9598',
    silver: '#c6cace',
    tomato: '#ea341f',
    darkGrey: '#262829',
    paleGrey: '#eef1f3',
    white: '#FFF',
    black30: 'rgba(0, 0, 0, .3)',
    pumpkinOrange: '#f58400',
    deepSkyBlue: '#1e75f6',
  }
}
