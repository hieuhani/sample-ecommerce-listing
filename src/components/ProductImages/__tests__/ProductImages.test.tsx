import React from 'react'
import renderer from 'react-test-renderer'
import { ThemeProvider } from 'styled-components'
import { ProductImages } from '..'
import { theme } from '../../../theme'

const props = {
  images: [{
    url: 'testurl',
    priority: 1,
  }]
}
describe('ProductImages', () => {
  test('ProductImages renders correctly', () => {
    const tree = renderer.create(
      <ThemeProvider theme={theme}>
        <ProductImages {...props} />
      </ThemeProvider>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
