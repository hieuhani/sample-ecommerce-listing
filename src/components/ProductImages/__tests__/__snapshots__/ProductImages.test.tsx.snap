// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImages ProductImages renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: #FFF;
}

.c1 {
  max-width: 330pt;
}

.c1 img {
  width: 100%;
}

<div
  className="swiper-container"
>
  <div
    className="swiper-wrapper"
  >
    <div
      className="c0 swiper-slide"
    >
      <div
        className="c1"
      >
        <img
          src="testurl"
        />
      </div>
    </div>
  </div>
  <div
    className="swiper-pagination"
  />
</div>
`;
